x-pack/plugins/ml/public/application/settings/calendars/edit/new_calendar.js

```diff
@@ -13,7 +13,7 @@ import { i18n } from '@kbn/i18n';
 import { EuiPageBody } from '@elastic/eui';
 
 import { getCalendarSettingsData, validateCalendarId } from './utils';
-import { CalendarForm } from './calendar_form';
+import { CalendarForm } from './calendar_form/calendar_form';
```

Comment:
Q: Why are we directly importing from the file instead?

Reply:
I think this is just a vscode autocomplete change. I'll change it back.